angularjs-1.5-basics/src/app/interval-example/interval-example.component.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+angular.module('interval-example')
```

Comment:
__[jshint]__
* 'angular' is not defined.

<!-- Comment by [SideCI](https://sideci.com) -->